src/System.Windows.Forms.Design/tests/UnitTests/DesignerFrameTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using Xunit;
+using Moq;
+
+namespace System.Windows.Forms.Design.Tests
+{
+    public class DesignerFrameTests
+    {
+        static private DesignerFrame CreateDesignerFrame()
+        {
+            var mockDictionary = new Dictionary<string, Color>();
+            mockDictionary.Add("ArtboardBackground", Color.White);
+
+            var mockIUIService = new Mock<IUIService>(MockBehavior.Default);
+            mockIUIService
+                .Setup(s => s.Styles)
+                .Returns(mockDictionary);
+
+            var mockSite = new Mock<ISite>(MockBehavior.Default);
+            mockSite
+                .Setup(s => s.GetService(typeof(IUIService)))
+                .Returns(mockIUIService.Object);
+
+            return new DesignerFrame(mockSite.Object);
+        }
+
+        [WinFormsFact]
+        public void DesignerFrame_Ctor_Default()
+        {
+            DesignerFrame designerFrame = CreateDesignerFrame();
```

Comment:
We can auto-dispose the control once we done with it:
```suggestion
            using DesignerFrame designerFrame = CreateDesignerFrame();
```
And we won't need to disposed it manually below